frontend/src/metabase/lib/schema_metadata.js

```diff
@@ -759,6 +759,7 @@ export const ICON_MAPPING = {
   [NUMBER]: "int",
   [BOOLEAN]: "io",
   [FOREIGN_KEY]: "connections",
+  [PRIMARY_KEY]: "key",
```

Comment:
I think that showing a key here is too literal for non-analyst folks, and would more likely make them think that this field is a password or access key of some kind.

I think a better choice would be to use our `label` icon because it looks like a tag — like a tag you'd find on a piece of clothing for sale that uniquely identifies that item.

`label` is unfortunately what we already use to denote fields with the `Category` type, but I think we could address that by adding a new icon for `Category` in a separate PR.

Reply:
@mazameli it makes sense, so let's not confuse non-technical users. For now, I've changed it to `label`, thanks! 
Could you please create an icon for Category type when you have time?